Tests/backend-specific-tests/Synapse/RunnerSynapseTest.php

```diff
@@ -227,22 +249,22 @@ public function testAbsWorkspaceMapping()
             'run',
             'run',
             '1234567',
-            new NullUsageFile(),
-            []
+            new NullUsageFile()
         );
         self::assertTrue($this->getContainerHandler()->hasInfoThatContains(sprintf('data/in/files/my_lovely_file.wtf/%s', $fileId)));
 
-        // assert the workspace is removed
+        // assert the workspace is preserved
```

Comment:
Preco je preserved a nie removed?

Reply:
Protože ABS workspaces jsou teď persistentní - to je tahle kulišárna https://keboola.atlassian.net/browse/KBCP-362